src/theme/Layout/index.js

```diff
@@ -12,6 +13,12 @@ import Footer from '@theme/Footer';
 
 import './styles.css';
 
+// Provided via plugins/react-axe-ada-monitoring
```

Comment:
Why is this in a theme? Are we looking to initiate `react-axe`? If so let's find somewhere outside the theme.

Reply:
@jjwill I don't know of anywhere else this could be put.. I could put conditional/initiation in an external file and require that here, but as far as I know all the js running on page load is called from theme files, and react-axe needs to run after the ReactDOM has initiated.
The plugin lifecycle happens at build/compile time

Reply:
Can we add a new component in the theme?

Reply:
We could but all that would mean is that we would require that component in a file like layout and invoke it... I don't see that as any cleaner since the code for react-axe is only like 3 lines

Reply:
There should be some way to import this w/ plugins. Any way to extend the main index? I don't want this to be in an extended component. this is something that belongs in a webpack config or index.js. It might be worth opening a ticket in docusaurus.

Reply:
I think all the plugins are build time... Do you talk with the docusuarus people still? If so do you mind bring this up and seeing if they know a way?